usecases/decision_workflows/decision_workflows.go

```diff
@@ -43,7 +43,7 @@ type caseAndDecisionRepository interface {
 type webhookEventCreator interface {
 	CreateWebhookEvent(
 		ctx context.Context,
-		tx repositories.Executor,
+		tx repositories.Transaction,
```

Comment:
"CreateWebhookEvent" should be used with a transaction.
Then, by transitivity so should "AutomaticDecisionToCase"